mavros_extras plugins not loaded with ROS2 Jazzy on Ubuntu 24.04

### Issue details

I build mavros from source according to the Jazzy instructions (commit b0da849a06eb1a215b9205b92b8cf39c6d7cf88f (HEAD, tag: 2.8.0, origin/ros2, origin/HEAD, ros2)), but upon running the mavros node, it fails to load all plugins in the mavros_extras, although the std_plugins are loaded successfully.


### MAVROS version and platform

Mavros: 2.8.0
ROS: Jazzy
Ubuntu: 24.04


### Autopilot type and version

[  ] ArduPilot
[  ] PX4

Version: N/A


### Node logs

```
[INFO] [1718454301.043908190] [mavros]: Plugin actuator_control created
[INFO] [1718454301.045180490] [mavros]: Plugin actuator_control initialized
[ERROR] [1718454301.076783145] [mavros]: Plugin adsb load exception: Failed to load library /home/igor.campos/ros2_ws/install/mavros_extras/lib/libmavros_extras_plugins.so. Make sure that you are calling the PLUGINLIB_EXPORT_CLASS macro in the library code, and that names are consistent between this macro and your XML. Error string: Could not load library dlopen error: /home/igor.campos/ros2_ws/install/mavros_extras/lib/libmavros_extras_plugins.so: undefined symbol: _ZTIN4YAML13BadConversionE, at ./src/shared_library.c:99
[INFO] [1718454301.102932325] [mavros]: Plugin altitude created
[INFO] [1718454301.103591558] [mavros]: Plugin altitude initialized
[ERROR] [1718454301.121035135] [mavros]: Plugin cam_imu_sync load exception: Failed to load library /home/igor.campos/ros2_ws/install/mavros_extras/lib/libmavros_extras_plugins.so. Make sure that you are calling the PLUGINLIB_EXPORT_CLASS macro in the library code, and that names are consistent between this macro and your XML. Error string: Could not load library dlopen error: /home/igor.campos/ros2_ws/install/mavros_extras/lib/libmavros_extras_plugins.so: undefined symbol: _ZTIN4YAML13BadConversionE, at ./src/shared_library.c:99
[ERROR] [1718454301.138585266] [mavros]: Plugin camera load exception: Failed to load library /home/igor.campos/ros2_ws/install/mavros_extras/lib/libmavros_extras_plugins.so. Make sure that you are calling the PLUGINLIB_EXPORT_CLASS macro in the library code, and that names are consistent between this macro and your XML. Error string: Could not load library dlopen error: /home/igor.campos/ros2_ws/install/mavros_extras/lib/libmavros_extras_plugins.so: undefined symbol: _ZTIN4YAML13BadConversionE, at ./src/shared_library.c:99
[ERROR] [1718454301.155941899] [mavros]: Plugin cellular_status load exception: Failed to load library /home/igor.campos/ros2_ws/install/mavros_extras/lib/libmavros_extras_plugins.so. Make sure that you are calling the PLUGINLIB_EXPORT_CLASS macro in the library code, and that names are consistent between this macro and your XML. Error string: Could not load library dlopen error: /home/igor.campos/ros2_ws/install/mavros_extras/lib/libmavros_extras_plugins.so: undefined symbol: _ZTIN4YAML13BadConversionE, at ./src/shared_library.c:99
[INFO] [1718454301.235567926] [mavros]: Plugin command created
[INFO] [1718454301.236171270] [mavros]: Plugin command initialized
[ERROR] [1718454301.253585588] [mavros]: Plugin companion_process_status load exception: Failed to load library /home/igor.campos/ros2_ws/install/mavros_extras/lib/libmavros_extras_plugins.so. Make sure that you are calling the PLUGINLIB_EXPORT_CLASS macro in the library code, and that names are consistent between this macro and your XML. Error string: Could not load library dlopen error: /home/igor.campos/ros2_ws/install/mavros_extras/lib/libmavros_extras_plugins.so: undefined symbol: _ZTIN4YAML13BadConversionE, at ./src/shared_library.c:99
[ERROR] [1718454301.270976054] [mavros]: Plugin debug_value load exception: Failed to load library /home/igor.campos/ros2_ws/install/mavros_extras/lib/libmavros_extras_plugins.so. Make sure that you are calling the PLUGINLIB_EXPORT_CLASS macro in the library code, and that names are consistent between this macro and your XML. Error string: Could not load library dlopen error: /home/igor.campos/ros2_ws/install/mavros_extras/lib/libmavros_extras_plugins.so: undefined symbol: _ZTIN4YAML13BadConversionE, at ./src/shared_library.c:99
[ERROR] [1718454301.288362169] [mavros]: Plugin distance_sensor load exception: Failed to load library /home/igor.campos/ros2_ws/install/mavros_extras/lib/libmavros_extras_plugins.so. Make sure that you are calling the PLUGINLIB_EXPORT_CLASS macro in the library code, and that names are consistent between this macro and your XML. Error string: Could not load library dlopen error: /home/igor.campos/ros2_ws/install/mavros_extras/lib/libmavros_extras_plugins.so: undefined symbol: _ZTIN4YAML13BadConversionE, at ./src/shared_library.c:99
[ERROR] [1718454301.305715728] [mavros]: Plugin esc_status load exception: Failed to load library /home/igor.campos/ros2_ws/install/mavros_extras/lib/libmavros_extras_plugins.so. Make sure that you are calling the PLUGINLIB_EXPORT_CLASS macro in the library code, and that names are consistent between this macro and your XML. Error string: Could not load library dlopen error: /home/igor.campos/ros2_ws/install/mavros_extras/lib/libmavros_extras_plugins.so: undefined symbol: _ZTIN4YAML13BadConversionE, at ./src/shared_library.c:99
[ERROR] [1718454301.323401284] [mavros]: Plugin esc_telemetry load exception: Failed to load library /home/igor.campos/ros2_ws/install/mavros_extras/lib/libmavros_extras_plugins.so. Make sure that you are calling the PLUGINLIB_EXPORT_CLASS macro in the library code, and that names are consistent between this macro and your XML. Error string: Could not load library dlopen error: /home/igor.campos/ros2_ws/install/mavros_extras/lib/libmavros_extras_plugins.so: undefined symbol: _ZTIN4YAML13BadConversionE, at ./src/shared_library.c:99
[ERROR] [1718454301.340907082] [mavros]: Plugin fake_gps load exception: Failed to load library /home/igor.campos/ros2_ws/install/mavros_extras/lib/libmavros_extras_plugins.so. Make sure that you are calling the PLUGINLIB_EXPORT_CLASS macro in the library code, and that names are consistent between this macro and your XML. Error string: Could not load library dlopen error: /home/igor.campos/ros2_ws/install/mavros_extras/lib/libmavros_extras_plugins.so: undefined symbol: _ZTIN4YAML13BadConversionE, at ./src/shared_library.c:99
[INFO] [1718454301.450236581] [mavros]: Plugin ftp created
[INFO] [1718454301.450928851] [mavros]: Plugin ftp initialized
[INFO] [1718454301.497113244] [mavros]: Plugin geofence created
[INFO] [1718454301.498707004] [mavros]: Plugin geofence initialized
[ERROR] [1718454301.516375318] [mavros]: Plugin gimbal_control load exception: Failed to load library /home/igor.campos/ros2_ws/install/mavros_extras/lib/libmavros_extras_plugins.so. Make sure that you are calling the PLUGINLIB_EXPORT_CLASS macro in the library code, and that names are consistent between this macro and your XML. Error string: Could not load library dlopen error: /home/igor.campos/ros2_ws/install/mavros_extras/lib/libmavros_extras_plugins.so: undefined symbol: _ZTIN4YAML13BadConversionE, at ./src/shared_library.c:99
[INFO] [1718454301.606536668] [mavros]: Plugin global_position created
[INFO] [1718454301.607774765] [mavros]: Plugin global_position initialized
[ERROR] [1718454301.625774354] [mavros]: Plugin gps_input load exception: Failed to load library /home/igor.campos/ros2_ws/install/mavros_extras/lib/libmavros_extras_plugins.so. Make sure that you are calling the PLUGINLIB_EXPORT_CLASS macro in the library code, and that names are consistent between this macro and your XML. Error string: Could not load library dlopen error: /home/igor.campos/ros2_ws/install/mavros_extras/lib/libmavros_extras_plugins.so: undefined symbol: _ZTIN4YAML13BadConversionE, at ./src/shared_library.c:99
[ERROR] [1718454301.643209171] [mavros]: Plugin gps_rtk load exception: Failed to load library /home/igor.campos/ros2_ws/install/mavros_extras/lib/libmavros_extras_plugins.so. Make sure that you are calling the PLUGINLIB_EXPORT_CLASS macro in the library code, and that names are consistent between this macro and your XML. Error string: Could not load library dlopen error: /home/igor.campos/ros2_ws/install/mavros_extras/lib/libmavros_extras_plugins.so: undefined symbol: _ZTIN4YAML13BadConversionE, at ./src/shared_library.c:99
[ERROR] [1718454301.660592397] [mavros]: Plugin gps_status load exception: Failed to load library /home/igor.campos/ros2_ws/install/mavros_extras/lib/libmavros_extras_plugins.so. Make sure that you are calling the PLUGINLIB_EXPORT_CLASS macro in the library code, and that names are consistent between this macro and your XML. Error string: Could not load library dlopen error: /home/igor.campos/ros2_ws/install/mavros_extras/lib/libmavros_extras_plugins.so: undefined symbol: _ZTIN4YAML13BadConversionE, at ./src/shared_library.c:99
[ERROR] [1718454301.677994585] [mavros]: Plugin guided_target load exception: Failed to load library /home/igor.campos/ros2_ws/install/mavros_extras/lib/libmavros_extras_plugins.so. Make sure that you are calling the PLUGINLIB_EXPORT_CLASS macro in the library code, and that names are consistent between this macro and your XML. Error string: Could not load library dlopen error: /home/igor.campos/ros2_ws/install/mavros_extras/lib/libmavros_extras_plugins.so: undefined symbol: _ZTIN4YAML13BadConversionE, at ./src/shared_library.c:99
[ERROR] [1718454301.695321071] [mavros]: Plugin hil load exception: Failed to load library /home/igor.campos/ros2_ws/install/mavros_extras/lib/libmavros_extras_plugins.so. Make sure that you are calling the PLUGINLIB_EXPORT_CLASS macro in the library code, and that names are consistent between this macro and your XML. Error string: Could not load library dlopen error: /home/igor.campos/ros2_ws/install/mavros_extras/lib/libmavros_extras_plugins.so: undefined symbol: _ZTIN4YAML13BadConversionE, at ./src/shared_library.c:99
[INFO] [1718454301.733895052] [mavros]: Plugin home_position created
[INFO] [1718454301.734573600] [mavros]: Plugin home_position initialized
[INFO] [1718454301.790885413] [mavros]: Plugin imu created
[INFO] [1718454301.792491673] [mavros]: Plugin imu initialized
[ERROR] [1718454301.811217698] [mavros]: Plugin landing_target load exception: Failed to load library /home/igor.campos/ros2_ws/install/mavros_extras/lib/libmavros_extras_plugins.so. Make sure that you are calling the PLUGINLIB_EXPORT_CLASS macro in the library code, and that names are consistent between this macro and your XML. Error string: Could not load library dlopen error: /home/igor.campos/ros2_ws/install/mavros_extras/lib/libmavros_extras_plugins.so: undefined symbol: _ZTIN4YAML13BadConversionE, at ./src/shared_library.c:99
[INFO] [1718454301.865069021] [mavros]: Plugin local_position created
[INFO] [1718454301.866100287] [mavros]: Plugin local_position initialized
[ERROR] [1718454301.884729035] [mavros]: Plugin log_transfer load exception: Failed to load library /home/igor.campos/ros2_ws/install/mavros_extras/lib/libmavros_extras_plugins.so. Make sure that you are calling the PLUGINLIB_EXPORT_CLASS macro in the library code, and that names are consistent between this macro and your XML. Error string: Could not load library dlopen error: /home/igor.campos/ros2_ws/install/mavros_extras/lib/libmavros_extras_plugins.so: undefined symbol: _ZTIN4YAML13BadConversionE, at ./src/shared_library.c:99
[ERROR] [1718454301.902142150] [mavros]: Plugin mag_calibration_status load exception: Failed to load library /home/igor.campos/ros2_ws/install/mavros_extras/lib/libmavros_extras_plugins.so. Make sure that you are calling the PLUGINLIB_EXPORT_CLASS macro in the library code, and that names are consistent between this macro and your XML. Error string: Could not load library dlopen error: /home/igor.campos/ros2_ws/install/mavros_extras/lib/libmavros_extras_plugins.so: undefined symbol: _ZTIN4YAML13BadConversionE, at ./src/shared_library.c:99
[INFO] [1718454301.945935848] [mavros]: Plugin manual_control created
[INFO] [1718454301.946588119] [mavros]: Plugin manual_control initialized
[ERROR] [1718454301.964550004] [mavros]: Plugin mocap_pose_estimate load exception: Failed to load library /home/igor.campos/ros2_ws/install/mavros_extras/lib/libmavros_extras_plugins.so. Make sure that you are calling the PLUGINLIB_EXPORT_CLASS macro in the library code, and that names are consistent between this macro and your XML. Error string: Could not load library dlopen error: /home/igor.campos/ros2_ws/install/mavros_extras/lib/libmavros_extras_plugins.so: undefined symbol: _ZTIN4YAML13BadConversionE, at ./src/shared_library.c:99
[ERROR] [1718454301.981900675] [mavros]: Plugin mount_control load exception: Failed to load library /home/igor.campos/ros2_ws/install/mavros_extras/lib/libmavros_extras_plugins.so. Make sure that you are calling the PLUGINLIB_EXPORT_CLASS macro in the library code, and that names are consistent between this macro and your XML. Error string: Could not load library dlopen error: /home/igor.campos/ros2_ws/install/mavros_extras/lib/libmavros_extras_plugins.so: undefined symbol: _ZTIN4YAML13BadConversionE, at ./src/shared_library.c:99
[INFO] [1718454302.017809613] [mavros]: Plugin nav_controller_output created
[INFO] [1718454302.018521345] [mavros]: Plugin nav_controller_output initialized
[ERROR] [1718454302.036250826] [mavros]: Plugin obstacle_distance load exception: Failed to load library /home/igor.campos/ros2_ws/install/mavros_extras/lib/libmavros_extras_plugins.so. Make sure that you are calling the PLUGINLIB_EXPORT_CLASS macro in the library code, and that names are consistent between this macro and your XML. Error string: Could not load library dlopen error: /home/igor.campos/ros2_ws/install/mavros_extras/lib/libmavros_extras_plugins.so: undefined symbol: _ZTIN4YAML13BadConversionE, at ./src/shared_library.c:99
[ERROR] [1718454302.053671866] [mavros]: Plugin odometry load exception: Failed to load library /home/igor.campos/ros2_ws/install/mavros_extras/lib/libmavros_extras_plugins.so. Make sure that you are calling the PLUGINLIB_EXPORT_CLASS macro in the library code, and that names are consistent between this macro and your XML. Error string: Could not load library dlopen error: /home/igor.campos/ros2_ws/install/mavros_extras/lib/libmavros_extras_plugins.so: undefined symbol: _ZTIN4YAML13BadConversionE, at ./src/shared_library.c:99
[ERROR] [1718454302.071015055] [mavros]: Plugin onboard_computer_status load exception: Failed to load library /home/igor.campos/ros2_ws/install/mavros_extras/lib/libmavros_extras_plugins.so. Make sure that you are calling the PLUGINLIB_EXPORT_CLASS macro in the library code, and that names are consistent between this macro and your XML. Error string: Could not load library dlopen error: /home/igor.campos/ros2_ws/install/mavros_extras/lib/libmavros_extras_plugins.so: undefined symbol: _ZTIN4YAML13BadConversionE, at ./src/shared_library.c:99
[ERROR] [1718454302.088541575] [mavros]: Plugin optical_flow load exception: Failed to load library /home/igor.campos/ros2_ws/install/mavros_extras/lib/libmavros_extras_plugins.so. Make sure that you are calling the PLUGINLIB_EXPORT_CLASS macro in the library code, and that names are consistent between this macro and your XML. Error string: Could not load library dlopen error: /home/igor.campos/ros2_ws/install/mavros_extras/lib/libmavros_extras_plugins.so: undefined symbol: _ZTIN4YAML13BadConversionE, at ./src/shared_library.c:99
[INFO] [1718454302.158863931] [mavros]: Plugin param created
[INFO] [1718454302.159493257] [mavros]: Plugin param initialized
[ERROR] [1718454302.177171442] [mavros]: Plugin play_tune load exception: Failed to load library /home/igor.campos/ros2_ws/install/mavros_extras/lib/libmavros_extras_plugins.so. Make sure that you are calling the PLUGINLIB_EXPORT_CLASS macro in the library code, and that names are consistent between this macro and your XML. Error string: Could not load library dlopen error: /home/igor.campos/ros2_ws/install/mavros_extras/lib/libmavros_extras_plugins.so: undefined symbol: _ZTIN4YAML13BadConversionE, at ./src/shared_library.c:99
[ERROR] [1718454302.194994293] [mavros]: Plugin px4flow load exception: Failed to load library /home/igor.campos/ros2_ws/install/mavros_extras/lib/libmavros_extras_plugins.so. Make sure that you are calling the PLUGINLIB_EXPORT_CLASS macro in the library code, and that names are consistent between this macro and your XML. Error string: Could not load library dlopen error: /home/igor.campos/ros2_ws/install/mavros_extras/lib/libmavros_extras_plugins.so: undefined symbol: _ZTIN4YAML13BadConversionE, at ./src/shared_library.c:99
[INFO] [1718454302.243961228] [mavros]: Plugin rallypoint created
[INFO] [1718454302.244570869] [mavros]: Plugin rallypoint initialized
[ERROR] [1718454302.262918102] [mavros]: Plugin rangefinder load exception: Failed to load library /home/igor.campos/ros2_ws/install/mavros_extras/lib/libmavros_extras_plugins.so. Make sure that you are calling the PLUGINLIB_EXPORT_CLASS macro in the library code, and that names are consistent between this macro and your XML. Error string: Could not load library dlopen error: /home/igor.campos/ros2_ws/install/mavros_extras/lib/libmavros_extras_plugins.so: undefined symbol: _ZTIN4YAML13BadConversionE, at ./src/shared_library.c:99
[INFO] [1718454302.321227023] [mavros]: Plugin rc_io created
[INFO] [1718454302.322475805] [mavros]: Plugin rc_io initialized
[INFO] [1718454302.372378378] [mavros]: Plugin setpoint_accel created
[INFO] [1718454302.372620967] [mavros]: Plugin setpoint_accel initialized
[INFO] [1718454302.457579544] [mavros]: Plugin setpoint_attitude created
[INFO] [1718454302.457764930] [mavros]: Plugin setpoint_attitude initialized
[INFO] [1718454302.532633901] [mavros]: Plugin setpoint_position created
[INFO] [1718454302.532839269] [mavros]: Plugin setpoint_position initialized
[INFO] [1718454302.620381056] [mavros]: Plugin setpoint_raw created
[INFO] [1718454302.621480340] [mavros]: Plugin setpoint_raw initialized
[INFO] [1718454302.690303583] [mavros]: Plugin setpoint_trajectory created
[INFO] [1718454302.690523451] [mavros]: Plugin setpoint_trajectory initialized
[INFO] [1718454302.750377595] [mavros]: Plugin setpoint_velocity created
[INFO] [1718454302.750611370] [mavros]: Plugin setpoint_velocity initialized
[INFO] [1718454302.900825194] [mavros]: Plugin sys_status created
[INFO] [1718454302.903548440] [mavros]: Plugin sys_status initialized
[INFO] [1718454302.953984822] [mavros.time]: TM: Timesync mode: MAVLINK
[INFO] [1718454302.974631806] [mavros]: Plugin sys_time created
[INFO] [1718454302.975544055] [mavros]: Plugin sys_time initialized
[ERROR] [1718454302.995014979] [mavros]: Plugin tdr_radio load exception: Failed to load library /home/igor.campos/ros2_ws/install/mavros_extras/lib/libmavros_extras_plugins.so. Make sure that you are calling the PLUGINLIB_EXPORT_CLASS macro in the library code, and that names are consistent between this macro and your XML. Error string: Could not load library dlopen error: /home/igor.campos/ros2_ws/install/mavros_extras/lib/libmavros_extras_plugins.so: undefined symbol: _ZTIN4YAML13BadConversionE, at ./src/shared_library.c:99
[ERROR] [1718454303.014797380] [mavros]: Plugin terrain load exception: Failed to load library /home/igor.campos/ros2_ws/install/mavros_extras/lib/libmavros_extras_plugins.so. Make sure that you are calling the PLUGINLIB_EXPORT_CLASS macro in the library code, and that names are consistent between this macro and your XML. Error string: Could not load library dlopen error: /home/igor.campos/ros2_ws/install/mavros_extras/lib/libmavros_extras_plugins.so: undefined symbol: _ZTIN4YAML13BadConversionE, at ./src/shared_library.c:99
[ERROR] [1718454303.033080744] [mavros]: Plugin trajectory load exception: Failed to load library /home/igor.campos/ros2_ws/install/mavros_extras/lib/libmavros_extras_plugins.so. Make sure that you are calling the PLUGINLIB_EXPORT_CLASS macro in the library code, and that names are consistent between this macro and your XML. Error string: Could not load library dlopen error: /home/igor.campos/ros2_ws/install/mavros_extras/lib/libmavros_extras_plugins.so: undefined symbol: _ZTIN4YAML13BadConversionE, at ./src/shared_library.c:99
[ERROR] [1718454303.051594308] [mavros]: Plugin tunnel load exception: Failed to load library /home/igor.campos/ros2_ws/install/mavros_extras/lib/libmavros_extras_plugins.so. Make sure that you are calling the PLUGINLIB_EXPORT_CLASS macro in the library code, and that names are consistent between this macro and your XML. Error string: Could not load library dlopen error: /home/igor.campos/ros2_ws/install/mavros_extras/lib/libmavros_extras_plugins.so: undefined symbol: _ZTIN4YAML13BadConversionE, at ./src/shared_library.c:99
[ERROR] [1718454303.069937856] [mavros]: Plugin vfr_hud load exception: Failed to load library /home/igor.campos/ros2_ws/install/mavros_extras/lib/libmavros_extras_plugins.so. Make sure that you are calling the PLUGINLIB_EXPORT_CLASS macro in the library code, and that names are consistent between this macro and your XML. Error string: Could not load library dlopen error: /home/igor.campos/ros2_ws/install/mavros_extras/lib/libmavros_extras_plugins.so: undefined symbol: _ZTIN4YAML13BadConversionE, at ./src/shared_library.c:99
[ERROR] [1718454303.088611252] [mavros]: Plugin vibration load exception: Failed to load library /home/igor.campos/ros2_ws/install/mavros_extras/lib/libmavros_extras_plugins.so. Make sure that you are calling the PLUGINLIB_EXPORT_CLASS macro in the library code, and that names are consistent between this macro and your XML. Error string: Could not load library dlopen error: /home/igor.campos/ros2_ws/install/mavros_extras/lib/libmavros_extras_plugins.so: undefined symbol: _ZTIN4YAML13BadConversionE, at ./src/shared_library.c:99
[ERROR] [1718454303.106788339] [mavros]: Plugin vision_pose load exception: Failed to load library /home/igor.campos/ros2_ws/install/mavros_extras/lib/libmavros_extras_plugins.so. Make sure that you are calling the PLUGINLIB_EXPORT_CLASS macro in the library code, and that names are consistent between this macro and your XML. Error string: Could not load library dlopen error: /home/igor.campos/ros2_ws/install/mavros_extras/lib/libmavros_extras_plugins.so: undefined symbol: _ZTIN4YAML13BadConversionE, at ./src/shared_library.c:99
[ERROR] [1718454303.125215867] [mavros]: Plugin vision_speed load exception: Failed to load library /home/igor.campos/ros2_ws/install/mavros_extras/lib/libmavros_extras_plugins.so. Make sure that you are calling the PLUGINLIB_EXPORT_CLASS macro in the library code, and that names are consistent between this macro and your XML. Error string: Could not load library dlopen error: /home/igor.campos/ros2_ws/install/mavros_extras/lib/libmavros_extras_plugins.so: undefined symbol: _ZTIN4YAML13BadConversionE, at ./src/shared_library.c:99
[INFO] [1718454303.201805669] [mavros]: Plugin waypoint created
[INFO] [1718454303.203036581] [mavros]: Plugin waypoint initialized
[ERROR] [1718454303.222686299] [mavros]: Plugin wheel_odometry load exception: Failed to load library /home/igor.campos/ros2_ws/install/mavros_extras/lib/libmavros_extras_plugins.so. Make sure that you are calling the PLUGINLIB_EXPORT_CLASS macro in the library code, and that names are consistent between this macro and your XML. Error string: Could not load library dlopen error: /home/igor.campos/ros2_ws/install/mavros_extras/lib/libmavros_extras_plugins.so: undefined symbol: _ZTIN4YAML13BadConversionE, at ./src/shared_library.c:99
[INFO] [1718454303.277355799] [mavros]: Plugin wind_estimation created
[INFO] [1718454303.278290806] [mavros]: Plugin wind_estimation initialized
```

--- a/README.md
+++ b/README.md
@@ -72,7 +72,8 @@ CI Statuses
 
   - ROS2 Humble: [![Build Status](https://build.ros2.org/job/Hdev__mavros__ubuntu_jammy_amd64/badge/icon)](https://build.ros2.org/job/Hdev__mavros__ubuntu_jammy_amd64/)
   - ROS2 Iron: [![Build Status](https://build.ros2.org/job/Idev__mavros__ubuntu_jammy_amd64/badge/icon)](https://build.ros2.org/job/Idev__mavros__ubuntu_jammy_amd64/)
-  - ROS2 Rolling: [![Build Status](https://build.ros2.org/job/Rdev__mavros__ubuntu_jammy_amd64/badge/icon)](https://build.ros2.org/job/Rdev__mavros__ubuntu_jammy_amd64/)
+  - ROS2 Jazzy: [![Build Status](https://build.ros2.org/job/Jdev__mavros__ubuntu_jammy_amd64/badge/icon)](https://build.ros2.org/job/Jdev__mavros__ubuntu_noble_amd64/)
+  - ROS2 Rolling: [![Build Status](https://build.ros2.org/job/Rdev__mavros__ubuntu_jammy_amd64/badge/icon)](https://build.ros2.org/job/Rdev__mavros__ubuntu_noble_amd64/)
 
 
 [mrrm]: https://github.com/mavlink/mavros/blob/master/mavros/README.md

